xdem/coreg/affine.py

```diff
@@ -1521,7 +1520,7 @@ def _fit_rst_pts(
 
         # Write output to class
         # (Mypy does not pass with normal dict, requires "OutAffineDict" here for some reason...)
-        output_affine = OutAffineDict(shift_x=-easting_offset, shift_y=-northing_offset, shift_z=vertical_offset)
+        output_affine = OutAffineDict(shift_x=easting_offset, shift_y=northing_offset, shift_z=vertical_offset)
```

Comment:
This line removes a temporary fix that was done in parallel of #595 to run this PR.
So there is actually no change at this line when comparing to `main`! (but you had it pop up when clicking on "See changes since last review")